packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -236,11 +236,22 @@ const styles: ComponentStyles<ISiteManagedClasses, IDevSiteDesignSystem> = {
     },
     site_transparencyToggleButton: {
         border: "none",
+        borderRadius: toPx(2),
         background: "none",
         height: toPx(32),
         width: toPx(32),
         cursor: "pointer",
-        outline: "0"
+        outline: "0",
+        marginRight: toPx(4),
+        opacity: ".85",
+        paddingTop: toPx(4),
+        "&:hover": {
+            opacity: "1"
+        },
+        "&[aria-pressed=\"true\"]": {
+            background: "#FFFFFF",
```

Comment:
Is there a reason that we are not using the design system here?

Reply:
Because it didn't work. You can see that the design system is used randomly in the file. Not sure the logic behind that, but neither: 
`background: (config: IDevSiteDesignSystem): string => config.backgroundColor,` or 
```
background: (config: IDevSiteDesignSystem): string => {
                return config.backgroundColor;
            }
```
render anything.

Reply:
Pulling this down and taking a look. It seems like something else is going on if this isn't working. If we can't figure it out, we can propose putting this in with hard coded values and file an issue, but I'd like us to first try and see what might be going on to cause this issue as it may not be a bug.

Reply:
@nalogiudice config.backgroundColor is coming in as undefined which is why it's not working. Can you quickly see if the other parts of this using the design system are also not showing up correctly? If they are working correctly it would be great to see if we are missing something as to why it's working sometimes in this file. If they are both not working, I think we should file a bug and note the bug in this file at the styles object. Then we can push this through

Reply:
@chrisdholt confirming that they all return `undefined`. Will add a bug

Reply:
[#694](https://github.com/Microsoft/fast-dna/issues/694)